tools/scripts/generate_simplified_db_schema.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+
+# Uncomment one of the imports depenging on the db that you want to generate simplified schema for
```

Comment:
No that's lazy. Just make it a normal script accepting a single argument. That should be db_type and have either `user` or `global` as a value.


```suggestion
# Uncomment one of the imports depending on the db that you want to generate simplified schema for
```